packages/client/tests/functional/relationMode/tests_m-to-n.ts

```diff
@@ -772,70 +806,50 @@ testMatrix.setupTestSuite(
               await prisma[categoryModel].findMany({
                 orderBy: { id: 'asc' },
               }),
-            ).toEqual(expectedFindManyCategoryModelIfNoChange)
+            ).toEqual(expectedFindManyCategoryModelIfUpdate)
             expect(await prisma[categoriesOnPostsModel].findMany({ orderBy: { categoryId: 'asc' } })).toEqual(
-              expectedFindManyCategoriesOnPostsModelIfNoChange,
+              expectedFindManyCategoriesOnPostsModelIfCategoryUpdate,
             )
           })
 
-          // For all databases (PostgreSQL, SQLite, MySQL, SQL Server, CockroachDB & MongoDB)
-          // onDelete: SetNull & relationMode: prisma
-          // fails the 2 following tests
-          // they are a copy above the tests above but with relationMode: prisma and `.failing`
-          // So we can run all the tests successfully
-          //
-          // For the first test:
-          // Received promise resolved instead of rejected
-          // Resolved to value: {"id": "3", "published": null}
-          //
-          // For the second test:
-          // Received promise resolved instead of rejected
-          // Resolved to value: {"id": "1-cat-a-updated", "published": null}
-          //
-          // See issue https://github.com/prisma/prisma/issues/15683
-
-          testIf(isRelationMode_prismaAndSetNull).failing(
+          testIf(isRelationMode_prismaAndSetNull)(
             'relationMode=prisma / SetNull: [update] post id should throw',
```

Comment:
```suggestion
            'relationMode=prisma / SetNull: [update] post id (optional relation) should succeed',
```
Looks like the test doesn't throw anymore, let's update the test names?

Reply:
Right, I have updated a bunch of names but it seems I forgot some. Thanks